src/Bootstrap/ButtonGroup.elm

```diff
@@ -57,12 +72,30 @@ type GroupItem msg
     = GroupItem (Html.Html msg)
 
 
-{-| Opaque type representing a button or link button, for composing button groups
+{-| Opaque type representing a button, for composing button groups
 -}
 type ButtonItem msg
     = ButtonItem (Html.Html msg)
 
 
+{-| Opaque type representing a link button, for composing button groups
+-}
+type LinkButtonItem msg
```

Comment:
Ah well, let's go with the way it is and see if anyone comes up with a valid use case for changing it !